
# Show Pointclouds
visiu_: true
console_output_: true

# Prefiltering parameter:
LEAF_SIZE_: 0.01
Z_LIM_MIN_: 0
Z_LIM_MAX_: 3.0    
Y_LIM_MIN_: -0.4
Y_LIM_MAX_: 0.17
CLUSTER_MIN_: 250 
CLUSTER_MAX_: 1100 #without robo 1000 
CLUSTER_TOLERANCE_: 0.04

# Shape filtering parameters:
PLANE_RATIO_: 0.4
ZYLINDER_RATIO_: 0.3
MAX_WIDTH_: 0.32 #without robo: 0.2
MAX_DEPTH_: 0.2

# Color filtering parameters:
YELLOW_RANGE_H_MIN_: 20.0
YELLOW_RANGE_H_MAX_: 70.0
YELLOW_RANGE_V_MIN_: 0.3
YELLOW_RANGE_V_MAX_: 1.0
YELLOW_RANGE_S_MIN_: 0.4
YELLOW_RANGE_S_MAX_: 1.0

GREEN_RANGE_H_MIN_: 72.0
GREEN_RANGE_H_MAX_: 170.0
GREEN_RANGE_V_MIN_: 0.2
GREEN_RANGE_V_MAX_: 1.0
GREEN_RANGE_S_MIN_: 0.2
GREEN_RANGE_S_MAX_: 1.0

BLUE_RANGE_H_MIN_: 220.0
BLUE_RANGE_H_MAX_: 250.0
BLUE_RANGE_V_MIN_: 0.1
BLUE_RANGE_V_MAX_: 1.0
BLUE_RANGE_S_MIN_: 0.1
BLUE_RANGE_S_MAX_: 0.5

DARK_RANGE_: 0.2

GREEN_RATIO_: 0.5
BLUE_RATIO_: 0.5
YELLOW_RATIO_: 0.45
DARK_RATIO_: 0.5

ROBO_HEIGHT_MIN_: 0.16
ROBO_HEIGHT_MAX_: 0.2
ROBO_WIDTH_MIN_: 0.25
ROBO_WIDTH_MAX_: 0.34

